packages/cli/index.js

```diff
@@ -12,19 +12,28 @@ const getRootDir = () => {
   return dirname(pkgFile);
 };
 
-try {
-  const rootDir = getRootDir();
-  const yargsPath = resolve(rootDir, '@untool/yargs');
-  require(yargsPath)
-    .configure({ mixins: [__dirname] })
-    .run();
-} catch (error) {
-  if (error.message && error.message.startsWith("Can't resolve")) {
-    // eslint-disable-next-line no-console
-    console.error("Error: Can't find @untool/yargs \n");
-  } else {
-    // eslint-disable-next-line no-console
-    console.error(error.stack ? error.stack.toString() : error.toString());
-  }
-  process.exit(1);
+exports.configure = (config = { mixins: [__dirname] }, options) => ({
```

Comment:
I'd like to have an actual function body here for readability..

Reply:
Also: I'm not too happy with the name `configure` - but don't know a better suggestion at the moment.